patch-package
--- a/node_modules/react-navigation-stack/dist/views/StackView/StackViewCard.js
+++ b/node_modules/react-navigation-stack/dist/views/StackView/StackViewCard.js
@@ -1,6 +1,6 @@
 import React from 'react';
 import { StyleSheet, Platform } from 'react-native';
-import { Screen } from './screens';
+import { Screen } from 'react-native-screens';
 import createPointerEventsContainer from './createPointerEventsContainer';
 
 const EPS = 1e-5;
--- a/node_modules/react-navigation-stack/dist/views/StackView/StackViewLayout.js
+++ b/node_modules/react-navigation-stack/dist/views/StackView/StackViewLayout.js
@@ -6,7 +6,7 @@ import { SceneView, StackActions, NavigationActions, withOrientation, Navigation
 
 import Card from './StackViewCard';
 import Header from '../Header/Header';
-import { ScreenContainer } from './screens';
+import { ScreenContainer } from 'react-native-screens';
 
 import TransitionConfigs from './StackViewTransitionConfigs';
 import { supportsImprovedSpringAnimation } from '../../utils/ReactNativeFeatures';
