patch-package
--- a/node_modules/react-navigation-fluid-transitions/FluidTransitioner.js
+++ b/node_modules/react-navigation-fluid-transitions/FluidTransitioner.js
@@ -2,7 +2,6 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { StyleSheet, Platform, Easing, I18nManager, Animated, PanResponder, InteractionManager } from 'react-native';
 import { NavigationActions, Transitioner } from 'react-navigation';
-import clamp from 'clamp';
 
 import TransitionItemsView from './TransitionItemsView';
 import TransitionRouteView from './TransitionRouteView';
@@ -20,6 +19,18 @@ type SceneRenderedInfo = {
   isMounted: boolean,
 };
 
+function clamp(number, lower, upper) {
+  if (number === number) {
+    if (upper !== undefined) {
+      number = number <= upper ? number : upper;
+    }
+    if (lower !== undefined) {
+      number = number >= lower ? number : lower;
+    }
+  }
+  return number;
+}
+
 class FluidTransitioner extends React.Component<*> {
   constructor(props) {
     super(props);
